### ISSUE TITLE
GDScript::load_byte_code trying to parse a wav file

### ISSUE THREAD

**Issue by noidexe** [godotengine/godot#16010]
**Godot version:**
3.0 rc-2

**OS/device including version:**
All platforms

**Issue description:**
Using preload() with .wav inside a gdscript files makes exported projects crash. 
Sample error log:
```
OpenGL ES 3.0 Renderer: Intel(R) HD Graphics 4000
GLES3: max ubo light: 409
GLES3: max ubo reflections: 455, ubo size: 144
ARVR: Registered interface: Native mobile
SCRIPT ERROR: GDScript::load_byte_code: Parse Error: Can't preload resource at path: res://sounds/1.wav
          At: res://scenes/avatar/Avatar.gdc:8
ERROR: GDScript::load_byte_code: Method/Function Failed, returning: ERR_PARSE_ERROR
   At: modules\gdscript\gdscript.cpp:764
ERROR: ResourceFormatLoaderGDScript::load: Condition ' err != OK ' is true. returned: RES()
   At: modules\gdscript\gdscript.cpp:1848
ERROR: Failed loading resource: res://scenes/avatar/Avatar.gdc
   At: core\io\resource_loader.cpp:186
ERROR: ResourceInteractiveLoaderText::poll: res://scenes/avatar/Avatar.tscn:3 - Parse Error: [ext_resource] referenced nonexistent resource at: res://scenes/avatar/Avatar.gd
   At: scene\resources\scene_format_text.cpp:439
ERROR: ResourceFormatLoader::load: Condition ' err != OK ' is true. returned: RES()
   At: core\io\resource_loader.cpp:149
ERROR: Failed loading resource: res://scenes/avatar/Avatar.tscn
   At: core\io\resource_loader.cpp:186
ERROR: ResourceInteractiveLoaderText::poll: res://scenes/timeline/Avatar-Boat.tscn:3 - Parse Error: [ext_resource] referenced nonexistent resource at: res://scenes/avatar/Avatar.tscn
   At: scene\resources\scene_format_text.cpp:439
ERROR: ResourceFormatLoader::load: Condition ' err != OK ' is true. returned: RES()
   At: core\io\resource_loader.cpp:149
ERROR: Failed loading resource: res://scenes/timeline/Avatar-Boat.tscn
   At: core\io\resource_loader.cpp:186
ERROR: ResourceInteractiveLoaderText::poll: res://scenes/dog/boat-body.tscn:4 - Parse Error: [ext_resource] referenced nonexistent resource at: res://scenes/timeline/Avatar-Boat.tscn
   At: scene\resources\scene_format_text.cpp:439
ERROR: ResourceFormatLoader::load: Condition ' err != OK ' is true. returned: RES()
   At: core\io\resource_loader.cpp:149
ERROR: Failed loading resource: res://scenes/dog/boat-body.tscn
   At: core\io\resource_loader.cpp:186
ERROR: ResourceInteractiveLoaderText::poll: res://scenes/timeline/Interaction-Boat.tscn:5 - Parse Error: [ext_resource] referenced nonexistent resource at: res://scenes/dog/boat-body.tscn
   At: scene\resources\scene_format_text.cpp:439
ERROR: ResourceFormatLoader::load: Condition ' err != OK ' is true. returned: RES()
   At: core\io\resource_loader.cpp:149
ERROR: Failed loading resource: res://scenes/timeline/Interaction-Boat.tscn
   At: core\io\resource_loader.cpp:186
ERROR: ResourceInteractiveLoaderText::poll: res://scenes/timeline/Timeline.tscn:6 - Parse Error: [ext_resource] referenced nonexistent resource at: res://scenes/timeline/Interaction-Boat.tscn
   At: scene\resources\scene_format_text.cpp:439
ERROR: ResourceFormatLoader::load: Condition ' err != OK ' is true. returned: RES()
   At: core\io\resource_loader.cpp:149
ERROR: Failed loading resource: res://scenes/timeline/Timeline.tscn
   At: core\io\resource_loader.cpp:186
ERROR: ResourceInteractiveLoaderText::poll: res://scenes/main/main.tscn:4 - Parse Error: [ext_resource] referenced nonexistent resource at: res://scenes/timeline/Timeline.tscn
   At: scene\resources\scene_format_text.cpp:439
ERROR: ResourceFormatLoader::load: Condition ' err != OK ' is true. returned: RES()
   At: core\io\resource_loader.cpp:149
ERROR: Failed loading resource: res://scenes/main/main.tscn
   At: core\io\resource_loader.cpp:186
ERROR: Failed loading scene: res://scenes/main/main.tscn
   At: main\main.cpp:1613
ERROR: SelfList<class GDScriptFunction>::List::~List: Condition ' _first != 0 ' is true.
   At: c:\projects\godot-builds\godot\core\self_list.h:100
ERROR: SelfList<class GDScript>::List::~List: Condition ' _first != 0 ' is true.
   At: c:\projects\godot-builds\godot\core\self_list.h:100
WARNING: ObjectDB::cleanup: ObjectDB Instances still exist!
     At: core\object.cpp:1989
ERROR: ResourceCache::clear: Resources Still in use at Exit!
   At: core\resource.cpp:418
```

The error doesn't happen when previewing inside the editor. 
It also stopped happening on export when removing the following lines:
```
var sounds = [
	preload("res://sounds/1.wav"),
	preload("res://sounds/3.wav"),
	preload("res://sounds/4.wav"),
	preload("res://sounds/6.wav")
]
```

**Comment by noidexe** [comment]
using load() instead of preload() doesn't trigger the error

**Comment by akien-mga** [comment]
Thanks for the report, that's a duplicate of the (fixed in master) #15902.
